x-pack/test/functional/apps/monitoring/elasticsearch/nodes.js

```diff
@@ -12,36 +12,204 @@ export default function ({ getService, getPageObjects }) {
   const nodesList = getService('monitoringElasticsearchNodes');
   const esClusterSummaryStatus = getService('monitoringElasticsearchSummaryStatus');
 
-  describe.skip('Elasticsearch nodes listing', () => { // Skip this because page takes too long to load. Fixed in 6.4.0+
```

Comment:
I unskipped this to see if the additional tests that have been added for this test will pass in automation. I confirmed that they run, and that they pass, but I need to go back and skip it again because this test is flaky in 6.3